resources/type-system/inference.md

```diff
@@ -1046,22 +1046,33 @@ The elaboration process sometimes introduces new operations that are not easily
 expressible using the syntax of Dart. To allow these operations to be specified
 succintly, the syntax of Dart is extended to allow the following forms:
 
-- `@AWAIT_WITH_TYPE_CHECK<T>(m_1)`, represents the following operation:
+- `@AWAIT_WITH_TYPE_CHECK(m_1)` represents the following operation:
 
-  - Evaluate `m_1`, and let `v` denote the result.
+  - Let `T_1` be the static type of `m_1`.
```

Comment:
I think the `-` needs to be indented by four spaces relative to the one above, to count as the next level of indentation.

Reply:
I don't believe that's necessary, based on Github's markdown preview (and based on the use of bulleted lists elsewhere in the file).